content/guides/references/error-messages.md

```diff
@@ -633,6 +633,10 @@ This error likely appeared because:
 1. You do not have internet. Please ensure you have connectivity then try again.
 2. You are a developer that has forked our codebase and do not have access to run our API locally. Please read more about this in our [contributing doc](https://on.cypress.io/contributing).
 
+### <Icon name="exclamation-triangle" color="red"></Icon> Whoops, we can't run your tests
+
+See [Cypress detected policy settings on your computer that may cause issues](#Cypress-detected-policy-settings-on-your-computer-that-may-cause-issues) below.
```

Comment:
I'd like to see us go into a little more depth here. Per @flotwig this error can be seen when any of the following occur:

* a policy blocks the extension/proxy settings
* a plugin breaks the `--proxy-server` or `--load-extension` arguments
* the user visits the Cypress proxy URL outside of a Cypress browser

The first bullet seems to be covered here, but not the other two.